node-api/backend/genesis.go

```diff
@@ -35,3 +35,31 @@ func (b Backend) GenesisValidatorsRoot(slot math.Slot) (common.Root, error) {
 	}
 	return st.GetGenesisValidatorsRoot()
 }
+
+// GenesisForkVersion returns the genesis fork version of the beacon chain.
+func (b Backend) GenesisForkVersion(genesisSlot math.Slot) (common.Version, error) {
```

Comment:
nit: all backend methods should have ptr receiver. Please change in another PR

Reply:
Created an issue https://github.com/berachain/beacon-kit/issues/2614 to track it. Will pick in separate PR